buildSrc/src/main/java/org/elasticsearch/gradle/info/GlobalBuildInfoPlugin.java

```diff
@@ -95,6 +95,9 @@ private static File findCompilerJavaHome() {
         String compilerJavaProperty = System.getProperty("compiler.java");
 
         if (compilerJavaProperty != null) {
+            if (compilerJavaProperty.contains("/")) {
```

Comment:
Note that this is temporary for testing this pull request. It will be removed before merging.